<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidgetMachineParams</class>
 <widget class="QWidget" name="SettingWidgetMachineParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="lineEdit_25">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lineEdit_22">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="lineEdit_21">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="lineEdit_24">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="lineEdit_23">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QLineEdit" name="lineEdit_34">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLineEdit" name="lineEdit_31">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_17">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QLineEdit" name="lineEdit_33">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLineEdit" name="lineEdit_26">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="8">
      <widget class="QLineEdit" name="lineEdit_37">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLineEdit" name="lineEdit_27">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="8">
      <widget class="QLineEdit" name="lineEdit_36">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_16">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QLineEdit" name="lineEdit_35">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QLineEdit" name="lineEdit_32">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_17">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>lineEdit_12</tabstop>
  <tabstop>lineEdit_13</tabstop>
  <tabstop>lineEdit_14</tabstop>
  <tabstop>lineEdit_15</tabstop>
  <tabstop>lineEdit_16</tabstop>
  <tabstop>lineEdit_17</tabstop>
  <tabstop>lineEdit_21</tabstop>
  <tabstop>lineEdit_22</tabstop>
  <tabstop>lineEdit_23</tabstop>
  <tabstop>lineEdit_24</tabstop>
  <tabstop>lineEdit_25</tabstop>
  <tabstop>lineEdit_26</tabstop>
  <tabstop>lineEdit_27</tabstop>
  <tabstop>lineEdit_31</tabstop>
  <tabstop>lineEdit_32</tabstop>
  <tabstop>lineEdit_33</tabstop>
  <tabstop>lineEdit_34</tabstop>
  <tabstop>lineEdit_35</tabstop>
  <tabstop>lineEdit_36</tabstop>
  <tabstop>lineEdit_37</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
